# inputs
int64 in
---
# outputs
int64 out
